.github/workflows/functional.yml

```diff
@@ -53,6 +53,9 @@ jobs:
   functional-das:
     name: "DAS"
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pytest-group: [ "1", "2", "3"]
```

Comment:
I have a few questions here:
1. how did you decide to use three groups
2. why do you need names for those names and how are those names used?

Reply:
1. 3 groups was a good balance considering the shortest time the tests take to run and the total time (sum of all jobs), but we can try other number of groups as well.
2. For `pytest-group: [ "1", "2", "3"]`: "1", "2", "3" are not used as names, but they are passed via `${{ matrix.pytest-group }}` to the test script as numbers and are used to define which testing group is running on every job. See https://jerry-git.github.io/pytest-split/#usage.